### DISCUSSION TITLE
Styling Tabs

### DISCUSSION THREAD

**Discussion by MPiccinato** [saadeghi/daisyui#2473] [Ideas / feature requests]
Hello! I am attempting to style the Tabs just a little, primarily the `tab-bordered` variant. This seems a little more difficult than it should be as I have had to make overrides in the theme such as...

```
          '.tab': {},
          '.tab-active:not(.tab-disabled):not([disabled])': {
            'color': 'hsl(var(--p))',
            'border-color': 'hsl(var(--p)) !important ',
          },
```

The idea/feature request would be to simply be able to apply another classname such as `tab-primary`, `tab-secondary` or `tab-accent` in order to get the right stylings applied depending on the variant being used.

**Comment by saadeghi** [comment]
I'm currently working on daisyUI 4.0  
This can be added in one of the minor updates after that 4.0 but I can't promise anything yet.   

However, you don't need to use `!important`.  
Your customization can get easier like this: https://play.tailwindcss.com/YRadgx4LXC  
Or this: https://play.tailwindcss.com/ADUEGechDN?file=config

**Reply by MPiccinato** [reply]
Thanks! The latter approach with `@apply` is much nicer.

I wouldn't mind taking a stab at an implementation if it would help.

**Reply by zyrm** [reply]
Hi @saadeghi. Has there been any progress on this?
